content/id/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -26,7 +26,7 @@ <h2>Panduan Dasar Kubernetes</h2>
         <p>Tutorial ini menyediakan panduan dasar mekanisme orkestrasi klaster Kubernetes. Setiap modul memliki beberapa informasi mengenai latar belakang bagi konsep mendasar dan <i>feature</i> Kubernetes, termasuk mode interaktif yang dapat digunakan sebagai metode pembelajaran <i>online</i>. Mode tutorial interaktif ini memberikan kesempatan pengguna untuk melakukan manajemen klaster sederhana beserta aplikasi dalam kontainer yang kamu miliki.</p>
```

Comment:
typo?
Use “fitur” https://kbbi.kemdikbud.go.id/entri/fitur
Keep “Container” because it is API name.
```suggestion
        <p>Tutorial ini menyediakan panduan dasar mekanisme orkestrasi klaster Kubernetes. Setiap modul memiliki beberapa informasi mengenai latar belakang bagi konsep mendasar dan fitur Kubernetes, termasuk mode interaktif yang dapat digunakan sebagai metode pembelajaran <i>online</i>. Mode tutorial interaktif ini memberikan kesempatan pengguna untuk melakukan manajemen klaster sederhana beserta aplikasi dalam Container yang kamu miliki.</p>
```

Reply:
The original document is not referencing to Container as an API Object. 

https://kubernetes.io/docs/tutorials/kubernetes-basics/

> These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.

```suggestion
        <p>Tutorial ini menyediakan panduan dasar mekanisme orkestrasi klaster Kubernetes. Setiap modul memliki beberapa informasi mengenai latar belakang bagi konsep mendasar dan fitur Kubernetes, termasuk mode interaktif yang dapat digunakan sebagai metode pembelajaran daring. Mode tutorial interaktif ini memberikan kesempatan pengguna untuk melakukan manajemen klaster sederhana beserta aplikasi dalam kontainer yang kamu miliki.</p>
```

Reply:
for “containerized” you can use “terkontainerisasi” later.

Reply:
```suggestion
        <p>Tutorial ini menyediakan panduan dasar mekanisme orkestrasi klaster Kubernetes. Setiap modul memliki beberapa informasi mengenai latar belakang bagi konsep mendasar dan <i>feature</i> Kubernetes, termasuk mode interaktif yang dapat digunakan sebagai metode pembelajaran <i>online</i>. Mode tutorial interaktif ini memberikan kesempatan pengguna untuk melakukan manajemen klaster sederhana beserta aplikasi terkontainerisasi yang kamu miliki.</p>
```